### DISCUSSION TITLE
Widen the peer dependency of @clr/angular to include Angular 15

### DISCUSSION THREAD

**Discussion by elgreco247** [vmware-clarity/ng-clarity#429] [General]
Hi, 

It would be nice to have the peer dependency of @clr/angular include Angular 15 to solve compatibility issues returned by NPM when installing.

Cheers!

**Comment by kevinbuhmann** [comment]
We have begun cutting v15 beta releases. https://github.com/vmware-clarity/ng-clarity/releases/tag/v15.0.0-beta.1
